src/Components/ServiceExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.DependencyInjection.Extensions;
+using System.Collections.Generic;
+
+namespace Components
+{
+	public static class ServiceExtensions
+	{
+		public static IServiceCollection UseComponentSystem(this IServiceCollection services)
+		{
+			services.TryAdd(DefaultServices());
```

Comment:
I think he's making the point that `DefaultServices` should be private/internal.